### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**

### CODE REVIEW

File: image/golem_blender_app/golem_blender_app/commands/verify.py [hunk 1]
```diff
@@ -58,7 +59,6 @@
         task_manager.update_subtask_status(
             subtask_id,
             SubtaskStatus.FAILURE)
-        # TODO: provide some extra info why verification failed
```
Comment (maaktweluit) [inline image/golem_blender_app/golem_blender_app/commands/verify.py]: i think this TODO is still valid

Comment (mfranciszkiewicz) [reply image/golem_blender_app/golem_blender_app/commands/verify.py]: True, there is still a need for the reason but it is not implemented in the verifier

Comment (maaktweluit) [reply image/golem_blender_app/golem_blender_app/commands/verify.py]: ok, so we want to remove it?

Comment (mfranciszkiewicz) [reply image/golem_blender_app/golem_blender_app/commands/verify.py]: I'll bring it back

**Comment by mfranciszkiewicz** [comment]
@maaktweluit Please review

**Pull request merged**